### PR TITLE
Holistic review on final_report

### PR DESCRIPTION
**Pull request by tonyshumlh**

### CODE REVIEW

**Review by jinyz8888 — COMMENTED** [review]
@tonyshumlh /analysis is in this branch again, do we need to remove it?

**Comment by tonyshumlh** [comment]
@jinyz8888 As I know, `analysis` is for storing the 30 run results and scripts

File: report/final_report/final_report.qmd [hunk 1]
```diff
@@ -18,9 +18,9 @@ by John Shiu, Orix Au Yeung, Tony Shum, Yingzi Jin
...
 
-However, ensuring the software quality of these systems remains a significant challenge [@openja2023studying]. Specifically, the lack of a standardized and comprehensive approach to testing machine learning (ML) systems introduces potential risks to stakeholders. For example, inadequate quality assurance in ML systems can lead to severe consequences, such as substantial financial losses ([@Asheeta2019], [@Asheeta2019], [@Asheeta2019]) and safety hazards. (FIXME: false information to clients, bias to society, substantial financial losses)
+However, ensuring the software quality of these systems remains a significant challenge ([@openja2023studying]). Specifically, the lack of a standardized and comprehensive approach to testing machine learning (ML) systems introduces potential risks to stakeholders. For example, inadequate quality assurance in ML systems can lead to severe consequences, such as misinformation ([@Ashley2024]), social bias ([@Alice2023]), substantial financial losses ([@Asheeta2019]) and safety hazards.
```
Comment (JohnShiuMK) [inline report/final_report/final_report.qmd:23]: I think 3 consequences are enough, as we seem not have references for safety hazards.
(but i may be able to find some)

**Comment by JohnShiuMK** [comment]
@tonyshumlh do we wanna review and merge #181 first?

File: report/final_report/final_report.qmd (hunk 1 above)
Comment (JohnShiuMK) [inline report/final_report/final_report.qmd:23]: I think 3 consequences are enough, as we seem not have references for safety hazards.
(but i may be able to find some)

**Review by JohnShiuMK — CHANGES_REQUESTED** [review]

File: report/final_report/final_report.qmd (hunk 1 above)
Comment (JohnShiuMK) [reply report/final_report/final_report.qmd:23]: https://tech.co/news/list-ai-failures-mistakes-errors

Comment (tonyshumlh) [reply report/final_report/final_report.qmd]: @JohnShiuMK a reference on safety hazard is added

**Pull request merged**